test/unit/uk/gov/hmrc/tariffclassificationfrontend/forms/FormDateTest.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.forms
+
+import java.time.{ZoneOffset, ZonedDateTime}
+
+import play.api.data.Form
+import uk.gov.hmrc.play.test.UnitSpec
+
+class FormDateTest extends UnitSpec {
+
+  val test = Form(FormDate.date())
+
+  "Date type" should {
+
+    "require fields" in {
+      test.bindFromRequest(
+        Map()
+      ).errors should have length 3
+    }
+
+    "disallow empty fields" in {
```

Comment:
Could we maybe have a test that shows it's an error if just one field was missed?